### PR TITLE
Event for pod admission once per pod

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:

Issue one event per pod during admission.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

Fix for unreleased feature.

```release-note
NONE
```

### CODE REVIEW

**Comment by alculquicondor** [comment]
While waiting for a review, trying to repro #1593

/test pull-kueue-test-e2e-main-1-27
/test pull-kueue-test-e2e-main-1-28
/test pull-kueue-test-e2e-main-1-29

**Comment by alculquicondor** [comment]
Updated

File: pkg/controller/jobframework/reconciler.go [hunk 1]
```diff
@@ -707,11 +708,9 @@ func (r *JobReconciler) startJob(ctx context.Context, job GenericJob, object cli
 		if err := r.client.Update(ctx, object); err != nil {
 			return err
 		}
+		r.record.Eventf(object, corev1.EventTypeNormal, ReasonStarted, msg)
```
[the remark this answers is not in the document]
Comment (tenzen-y) [reply pkg/controller/jobframework/reconciler.go:711]: @alculquicondor This is still remaining.

**Review by tenzen-y — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Comment by tenzen-y** [comment]
Another flaky test happened :(

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_kueue/1604/pull-kueue-test-integration-main/1747700693520617472

**Comment by alculquicondor** [comment]
I'll open another issue for it

**Comment by tenzen-y** [comment]
> I'll open another issue for it

Another seems to be similar to https://github.com/kubernetes-sigs/kueue/issues/1484.

**Comment by alculquicondor** [comment]
Opened #1605

I think it's quite different.

**Pull request merged**